Makefile

```diff
@@ -86,7 +85,8 @@ fmt:
 
 lint:
 	@echo "+ $@"
-	@test -z "$$(golint ./... | grep -v .pb. | grep -v Godeps/_workspace/src/ | tee /dev/stderr)"
+	# TODO: remove trustmanager/yubikey/ from ignored once linted
```

Comment:
just didn't want to overload this PR (plus I'm not so confident in documenting that right now)